routing/router.go

```diff
@@ -214,6 +214,10 @@ type ChannelPolicy struct {
 	// TimeLockDelta is the required HTLC timelock delta to be used
 	// when forwarding payments.
 	TimeLockDelta uint32
+
+	// MaxHTLC is the maximum HTLC size including fees we are allowed to
+	// forward over this channel.
+	MaxHTLC lnwire.MilliSatoshi
```

Comment:
Note in the `godoc` that it is optional?

Reply:
Left out the "optional" since this struct is only used in `rpcserver` and `gossiper`, and `MaxHtlc` will always be set by one of them (either the user in `UpdateChannelPolicy` or the `gossiper` if the user leaves `MaxHtlc` unset).

Reply:
Optional is still in the comment.